Forms/MainForm.cs

```diff
@@ -419,6 +419,30 @@ private void mItemIncl_Click(object sender, EventArgs e)
         }
 
 
+        private void cClockSpeed_SelectedIndexChanged(object sender, EventArgs e)
+        {
+            byte clockSpeed;
+            switch (cClockSpeed.SelectedIndex)
+            {
+                default:
+                case 0: // Default
+                    clockSpeed = 3;
+                    break;
+                case 1: // Slow
+                    clockSpeed = 2;
+                    break;
+                case 2: // Super slow
+                    clockSpeed = 1;
+                    break;
+                case 3: // Fast
+                    clockSpeed = 6;
+                    break;
+                case 4: // Super fast
+                    clockSpeed = 18;
+                    break;
+            }
+            UpdateSingleSetting(() => _settings.ClockSpeed = clockSpeed);

```

Comment:
I think it'd be better to save the selected index into the settings string, instead of the actual clockspeed value. Otherwise you're not limited to the choices in the dropdown, you can just put `-255` as the end of your settings string and it'll accept it. I don't think we want that do we?

Reply:
Agreed, will change it later.